### PR TITLE
converted multi-label-classification to kerascore

### PR DESCRIPTION
**Pull request by Akorex**
Port the [Large-scale multi-label classification](https://keras.io/examples/nlp/multi_label_classification) to Keras Core. 

Colab Notebook: https://colab.research.google.com/drive/1kR7W4BJhtDlA5Jx_e4EwUM2FnzKP-D58

### CODE REVIEW

File: examples/keras_io/tensorflow/nlp/multi_label_classification.py [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+
+"""## Multi-label binarization
+
+Now we preprocess our labels using the
+[`StringLookup`](https://keras.io/api/layers/preprocessing_layers/categorical/string_lookup)
+layer.
+"""
+
+terms = tf.ragged.constant(train_df["terms"].values)
+lookup = tf.keras.layers.StringLookup(output_mode="multi_hot")
```
Comment (fchollet) [inline examples/keras_io/tensorflow/nlp/multi_label_classification.py:148]: There should be no tf.keras.

File: examples/keras_io/tensorflow/nlp/multi_label_classification.py [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+
+## Imports
+"""
+
+
+!pip install -q git+https://github.com/keras-team/keras-core
+
+import keras_core as keras
+from keras_core import layers
+import tensorflow as tf
```
Comment (fchollet) [inline examples/keras_io/tensorflow/nlp/multi_label_classification.py:39]: Use `os.environ` to set the backend before importing.

File: examples/keras_io/tensorflow/nlp/multi_label_classification.py [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+
+from sklearn.model_selection import train_test_split
```
Comment (fchollet) [inline examples/keras_io/tensorflow/nlp/multi_label_classification.py:41]: You can use `split_dataset` for this.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR! Please post the git diff from the original file.

File: examples/keras_io/tensorflow/nlp/multi_label_classification.py (hunk 3 above)
Comment (Akorex) [reply examples/keras_io/tensorflow/nlp/multi_label_classification.py:41]: Yes, but the original example stratified the split on a column because of class imbalance. `split_dataset` to my knowledge isn't supported for this.

Comment (Akorex) [reply examples/keras_io/tensorflow/nlp/multi_label_classification.py:41]: Yes, but the original example stratified the split on a column because of class imbalance. `split_dataset` to my knowledge isn't supported for this.

Comment (fchollet) [reply examples/keras_io/tensorflow/nlp/multi_label_classification.py:41]: Ok

File: examples/keras_io/tensorflow/nlp/multi_label_classification.py (hunk 1 above)
Comment (fchollet) [reply examples/keras_io/tensorflow/nlp/multi_label_classification.py:148]: Please fix.
